### PR TITLE
LPD-35779 Add functional tests and instance scoped globalJS sample

### PR DESCRIPTION
**Pull request by thiago-buarqque**
Ticket: https://liferay.atlassian.net/browse/LPD-35779

### Added a new globalJS client extension sample.
That one is not showing up a window alert (as the other two samples) because it would mess up with other tests, since it is applied on every page. So I'm just logging something.

### Test scenario added

Assert that the GlobalJS sample 3 is injected into site pages, instance control panel pages and site control panel pages.

### CODE REVIEW

**Comment by thiago-buarqque** [comment]
ci:test:playwright

File: modules/test/playwright/tests/client-extension-web/js.spec.ts [hunk 1]
```diff
@@ -29,6 +31,11 @@ const SAMPLES = [
 		name: 'Liferay Sample Global JS 2',
 		url: '/o/liferay-sample-global-js-2/global.a0ff2e4a08889609cd1e.js',
 	},
+	{
+		erc: 'LXC:liferay-sample-global-js-3',
+		name: 'Liferay Sample Global JS 3',
+		url: '/o/liferay-sample-global-js-3/global.8c51c23d2fbc94a8abfa.js',
```
Comment (ethib137) [inline modules/test/playwright/tests/client-extension-web/js.spec.ts:37]: Hey @thiago-buarqque , won't the hash in this file name change everytime we deploy the CX?

**Comment by ethib137** [comment]
ci:test:playwright

File: modules/test/playwright/tests/client-extension-web/js.spec.ts (hunk 1 above)
Comment (thiago-buarqque) [reply modules/test/playwright/tests/client-extension-web/js.spec.ts:37]: @ethib137 No, that hash is built based on the file content. If you scroll up a little bit you'll see we have a fixed hash for the samples 1 and 2 as well

**Comment by thiago-buarqque** [comment]
ci:test:stable

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by thiago-buarqque** [comment]
Playwright errors are unrelated. Waiting for https://github.com/liferay-frontend/liferay-portal/pull/4416 to be merged and then I will forward this one

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by thiago-buarqque** [comment]
ci:test:stable

**Comment by thiago-buarqque** [comment]
ci:test:playwright

**Comment by thiago-buarqque** [comment]
Unrelated CI errors, @ethib137 is this one good to be forwarded?

**Comment by ethib137** [comment]
Forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/154236

@thiago-buarqque please see additional changes: https://github.com/brianchandotcom/liferay-portal/pull/154236/files/0c192e2a736761562ae1fd9be85d765c8b5e9818..77aacaf3b2d2daec2c00df2e7665ed4f7f603d63

**Comment by thiago-buarqque** [comment]
Thanks @ethib137

**Pull request closed without merging**